pwa/src/templates/templateParts/userForm/UserFormTemplate.tsx

```diff
@@ -55,6 +60,7 @@ export const UserFormTemplate: React.FC<UserFormTemplateProps> = ({ user }) => {
     const payload = {
       ...data,
       organisation: data.organization && `/admin/organisations/${data.organization.value}`,
+      applications: data.applications?.map((application: any) => `/admin/applications/${application.value}`),
```

Comment:
I think you can type this as a `IKeyValue` type, which is exported from @conduction/components.